iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/NetworkXml.java

```diff
@@ -618,140 +467,32 @@ public static Network read(ReadOnlyDataSource dataSource, NetworkFactory network
-                    try (InputStream ise = dataSource.newInputStream("-ext", dataSourceExt)) {
-                        readExtensions(network, ise, anonymizer, options);
-                    } catch (IOException e) {
-                        LOGGER.warn(String.format("the extensions file wasn't found while importing, please ensure that the file name respect the naming convention baseFileName-ext.%s", dataSourceExt));
-                    }
-                    break;
-                case ONE_SEPARATED_FILE_PER_EXTENSION_TYPE:
-                    String ext = dataSourceExt.isEmpty() ? "" : "." + dataSourceExt;
-                    // here we'll read all extensions declared in the extensions set
-                    readExtensions(network, dataSource, anonymizer, options, ext);
-                    break;
-                default:
-                    break;
-            }
-        }
         return network;
     }
 
-    public static Network read(Path xmlFile, ImportOptions options) throws IOException {
-        DataSource dataSource = getDataSourceFromPath(xmlFile);
-        String ext = getFileExtensionFromPath(xmlFile);
-        return read(dataSource, NetworkFactory.findDefault(), options, ext);
-    }
-
-    public static Network validateAndRead(Path xmlFile) {
-        try {
-            return validateAndRead(xmlFile, new ImportOptions());
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
-    }
-
-    private static void validate(Path xmlFile, IidmImportExportMode mode) throws IOException {
-        DataSource dataSource = getDataSourceFromPath(xmlFile);
-        String ext = getFileExtensionFromPath(xmlFile);
-
-        if (mode == IidmImportExportMode.EXTENSIONS_IN_ONE_SEPARATED_FILE) {
-            try (InputStream ise = dataSource.newInputStream(dataSource.getBaseName() + ext)) {
-                validateWithExtensions(ise);
-            }
-            try (InputStream ise = dataSource.newInputStream(dataSource.getBaseName() + "-ext." + ext)) {
-                validateWithExtensions(ise);
-            }
-        } else {
-            Set<String> listNames = dataSource.listNames(".*\\." + ext);
-            for (String fileName : listNames) {
-                try (InputStream ise = dataSource.newInputStream(fileName)) {
-                    validateWithExtensions(ise);
-                }
-            }
-        }
-    }
-
-    public static Network validateAndRead(Path xmlFile, ImportOptions options) throws IOException {
-        if (options.getMode() == IidmImportExportMode.UNIQUE_FILE) {
-            validateWithExtensions(xmlFile);
-            return read(xmlFile);
-        }
-        validate(xmlFile, options.getMode());
-        return read(xmlFile, options);
+    /**
+     * @deprecated Use {@link #read(Path)} instead.
+     */
+    @Deprecated
+    public static Network read(Path xmlFile, ImportOptions options) {
+        return read(xmlFile);
```

Comment:
Is it a good idea to forget the Import/export Options? Are you sure we should not keep both? As the ImportOptions is not deprecated, why this method is?